src/renderer/components/button/button.tsx

```diff
@@ -49,7 +50,11 @@ export const Button = withTooltip((props: ButtonProps) => {
 
   // render as button
   return (
-    <button type="button" {...btnProps}>
+    <button
+      type="button"
+      {...btnProps}
+      data-waiting={`${waiting}`}
```

Comment:
Is this necessary? Dialog button can be disabled with native `disabled` prop.

Reply:
This is not really for having it be disabled but instead being able to write a test for the button to show that it is in the loading state. Similar to other tests that makes sure that a spinner is shown while something is happening.

Reply:
I will move this change to a separate PR to make it easier to review

Reply:
I have made https://github.com/lensapp/lens/pull/6040